test/test_torch.py

```diff
@@ -15694,6 +15703,7 @@ def run_subtest(matrix_size, batches, device, pivot, singular=False, a=None):
                     rows, columns = matrix_size
                 if a is None:
                     a = random_matrix(rows, columns, *batches, **dict(singular=singular, dtype=dtype)).to(device)
+                a_ = a.cpu().numpy()
```

Comment:
I don't see where a_ is used - maybe github is not rendering.

Reply:
You are right, it is not used

Reply:
fixed